packages/app/obojobo-repository/shared/components/pages/page-module-hoc.js

```diff
@@ -0,0 +1,5 @@
+const Module = require('./page-module.jsx')
+const connect = require('react-redux').connect
+const mapStoreStateToProps = state => state
```

Comment:
I suspect we don't even need redux at all for this (though we may in the future). Can this be simplified a bit by removing the hoc?

Reply:
Done